charmap.asm

```diff
@@ -1,265 +1,386 @@
+	charmap "ギ", $06
+	charmap "グ", $07
+	charmap "ゲ", $08
+	charmap "ゴ", $09
+	charmap "ザ", $0a
+	charmap "ジ", $0b
+	charmap "ズ", $0c
+	charmap "ゼ", $0d
+	charmap "ゾ", $0e
+	charmap "ダ", $0f
+	charmap "ヂ", $10
+	charmap "ヅ", $11
+	charmap "デ", $12
+	charmap "ド", $13
+
+	charmap "バ", $19
+	charmap "ビ", $1a
+	charmap "ブ", $1b
+	charmap "ボ", $1c
+
+	charmap "が", $26
+	charmap "ぎ", $27
+	charmap "ぐ", $28
+	charmap "げ", $29
+	charmap "ご", $2a
+	charmap "ざ", $2b
+	charmap "じ", $2c
+	charmap "ず", $2d
+	charmap "ぜ", $2e
+	charmap "ぞ", $2f
+	charmap "だ", $30
+	charmap "ぢ", $31
+	charmap "づ", $32
+	charmap "で", $33
+	charmap "ど", $34
+
+	charmap "ば", $3a
+	charmap "び", $3b
+	charmap "ぶ", $3c
+	charmap "べ", $3d
+	charmap "ぼ", $3e
+
+	charmap "パ", $40
+	charmap "ピ", $41
+	charmap "プ", $42
+	charmap "ポ", $43
+	charmap "ぱ", $44
+	charmap "ぴ", $45
+	charmap "ぷ", $46
+	charmap "ぺ", $47
+	charmap "ぽ", $48
+
+	charmap "「", $70
+	charmap "」", $71
+	charmap "』", $73
+	charmap "⋯", $75
+
+	charmap "　", $7f
+
+	charmap "ア", $80
+	charmap "イ", $81
+	charmap "ウ", $82
+	charmap "エ", $83
+	charmap "オ", $84
+	charmap "カ", $85
+	charmap "キ", $86
+	charmap "ク", $87
+	charmap "ケ", $88
+	charmap "コ", $89
+	charmap "サ", $8a
+	charmap "シ", $8b
+	charmap "ス", $8c
+	charmap "セ", $8d
+	charmap "ソ", $8e
+	charmap "タ", $8f
+	charmap "チ", $90
+	charmap "ツ", $91
+	charmap "テ", $92
+	charmap "ト", $93
+	charmap "ナ", $94
+	charmap "ニ", $95
+	charmap "ヌ", $96
+	charmap "ネ", $97
+	charmap "ノ", $98
+	charmap "ハ", $99
+	charmap "ヒ", $9a
+	charmap "フ", $9b
+	charmap "ホ", $9c
+	charmap "マ", $9d
+	charmap "ミ", $9e
+	charmap "ム", $9f
+	charmap "メ", $a0
+	charmap "モ", $a1
+	charmap "ヤ", $a2
+	charmap "ユ", $a3
+	charmap "ヨ", $a4
+	charmap "ラ", $a5
+	charmap "ル", $a6
+	charmap "レ", $a7
+	charmap "ロ", $a8
+	charmap "ワ", $a9
+	charmap "ヲ", $aa
+	charmap "ン", $ab
+	charmap "ッ", $ac
+	charmap "ャ", $ad
+	charmap "ュ", $ae
+	charmap "ョ", $af
+	charmap "ィ", $b0
+
+	charmap "あ", $b1
+	charmap "い", $b2
+	charmap "う", $b3
+	charmap "え", $b4
+	charmap "お", $b5
+	charmap "か", $b6
+	charmap "き", $b7
+	charmap "く", $b8
+	charmap "け", $b9
+	charmap "こ", $ba
+	charmap "さ", $bb
+	charmap "し", $bc
+	charmap "す", $bd
+	charmap "せ", $be
+	charmap "そ", $bf
+	charmap "た", $c0
+	charmap "ち", $c1
+	charmap "つ", $c2
+	charmap "て", $c3
+	charmap "と", $c4
+	charmap "な", $c5
+	charmap "に", $c6
+	charmap "ぬ", $c7
+	charmap "ね", $c8
+	charmap "の", $c9
+	charmap "は", $ca
+	charmap "ひ", $cb
+	charmap "ふ", $cc
+	charmap "へ", $cd
+	charmap "ほ", $ce
+	charmap "ま", $cf
+	charmap "み", $d0
+	charmap "む", $d1
+	charmap "め", $d2
+	charmap "も", $d3
+	charmap "や", $d4
+	charmap "ゆ", $d5
+	charmap "よ", $d6
+	charmap "ら", $d7
+	charmap "り", $d8
+	charmap "る", $d9
+	charmap "れ", $da
+	charmap "ろ", $db
+	charmap "わ", $dc
+	charmap "を", $dd
+	charmap "ん", $de
+	charmap "っ", $df
+	charmap "ゃ", $e0
+	charmap "ゅ", $e1
+	charmap "ょ", $e2
+
+	charmap "ー", $e3
+	charmap "ﾟ", $e4
+	charmap "ﾞ", $e5
+
+	charmap "？", $e6
+	charmap "！", $e7
```

Comment:
Is there value to having multiple variations of an exclamation point when they have the exact same value? I understand that this copy is specifically for untranslated text, but it seems like it only creates an almost unnoticeable difference.

Reply:
There are a couple untranslated Japanese strings that use exclamation points, and IMO they look weird with standard-width kana and then a narrow ASCII "!". The fullwidth space and punctuation will likely use the same font as other Japanese characters.

Reply:
Okay, makes sense 👍